### ISSUE TITLE
12/4/2020 latest version

### ISSUE THREAD

**Issue by tharris9d** [fHDHR/fHDHR_Locast#113]
On Windows 10 with Threading.....
I restarted all apps and it's now all working. How can I best use the clustering?

**Comment by deathbybandaid** [comment]
The clustering is really easy.

Make sure `discovery_address` is set to something other than "0.0.0.0" (on all instances of fHDHR).

Make sure they are on the same network, SSDP might get tricky accross VLANs and whatnot.

Go to the Cluster/SSDP tab in the web interface, and click scan. If it sees other fHDHR devices, it will show a list of devices to connect to.

Clicking "Add" will send join information to the other fHDHR instance and Join them together.

Once that's complete, you'll see a second row of buttons under the main tabs, this will allow you quick switching between your instances.

**Comment by deathbybandaid** [comment]
Also of interest, try out the new CSS web UI tweaking!

https://github.com/fHDHR/CSS_Themes_for_fHDHR

Simply apply the url of a CSS file, and you can theme fHDHR.

I put together one that has a Plex feel to it, and am open to PRs for more themes

**Comment by deathbybandaid** [comment]
I'm debating a seperate fHDHR to pull all the channels from a cluster and manage the channel numbers in one place. However, xteve does this really well already, so it's not a priority.

**Comment by tharris9d** [comment]
I applied the css file through the settings ui but it's not working. I restarted the app and cleared the chrome cache. Any tips?

I have clustering working now. Thanks. 
I love the idea of a separate fHDHR to pull in all the channels from a cluster. xteve has it's issues. Your code works great. When I pull it into xteve I am constantly having to update the xmltv files, clearing the cache and uploading again. I don't have to do any of that if fHDHR_PlutoTV is set in Plex directly. Most of my issues are xteve related and not fHDHR. You have my vote!
